apps.awesim.org/apps/myjobs/templates/Basic_NAMD_Parallel_Job_Pitzer-Exp/slurm_script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#SBATCH -J ondemand/sys/myjobs/basic_namd_parallel
+#SBATCH --nodes=2
+#SBATCH --time=00:12:00
+#SBATCH --exclusive
+
+#   A Basic NAMD Parallel Job for the OSC Pitzer Cluster
+#   https://www.osc.edu/resources/available_software/software_list/namd
+
+#
+# The following lines set up the NAMD environment
+#
+module load intel/18.0.3  
```

Comment:
Why use older versions of modules? There is `intel/19.0.5` and also `mvapich2/2.3.4` that are what is used by default IIRC.